docs/source/algorithms/LoadILLIndirect-v2.rst

```diff
@@ -9,13 +9,19 @@
 Description
 -----------
 
-This algorithm loads a `.nxs` file produced by IN16B back-scattering spectrometer at ILL.
-It supports both QENS and FWS types of data, both with or without mirror sense.
+This is the specialized loader for the raw `.nxs` files produced by IN16B back-scattering spectrometer at ILL.
+This loader can load only a single file at each call. If loading more than one file is required, please refer to
+:ref:`Load <algm-Load>` or :ref:`LoadAndMerge <algm-LoadAndMerge>` algorithms, which are more suited to that task.
+
+
+The loader supports both QENS and FWS types of data, both with or without mirror sense.
 The number of active single detectors, and their actual positions are read from the `.nxs` file overriding the ones in the instrument definition.
 The output workspace will be dimensionless containing the number of counts per energy channel for each detector pixel.
 It will have `empty` x-axis unit.
+
 The `LoadDetectors` allows to choose which data to load for IN16B: those acquired by the spectrometer,
```

Comment:
```suggestion
The **LoadDetectors** option allows to choose which data to load for IN16B: those acquired by the spectrometer,
```

To make refferencing this function parameter in-line with the change made to ``LoadILLTOF-v2``